### PR TITLE
sdcicd-799 utilize the ensurePassingXML functionality in operator suite

### PR DESCRIPTION
**Pull request by ritmun**
This change will use the newer RetrieveTestResults function instead of the older RetrieveResults function which did not utilize ensurePassingXML which validates junit xml. 

This function is used in some newer suites but not retrofitted into existing ones, such as this (Operators) one. We should probably do that. 

jira https://issues.redhat.com/browse/SDCICD-799

### CODE REVIEW

File: pkg/e2e/operators/operators.go [hunk 1]
```diff
@@ -391,7 +391,7 @@ Loop:
 			return err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s clusterRoleBinding to exist", (timeoutDuration - elapsed), clusterRoleBindingName)
+				log.Printf("Waiting %v for %s clusterRoleBinding to exist", timeoutDuration-elapsed, clusterRoleBindingName)
```
Comment (MrSantamaria) [inline pkg/e2e/operators/operators.go:394]: Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.

File: pkg/e2e/operators/operators.go [hunk 2]
```diff
@@ -432,7 +432,7 @@ Loop:
 			return err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s roleBinding to exist", (timeoutDuration - elapsed), roleBindingName)
+				log.Printf("Waiting %v for %s roleBinding to exist", timeoutDuration-elapsed, roleBindingName)
```
Comment (MrSantamaria) [inline pkg/e2e/operators/operators.go:435]: Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.

File: pkg/e2e/operators/operators.go [hunk 3]
```diff
@@ -474,7 +474,7 @@ Loop:
 			return err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s configMap to exist", (timeoutDuration - elapsed), operatorLockFile)
+				log.Printf("Waiting %v for %s configMap to exist", timeoutDuration-elapsed, operatorLockFile)
```
Comment (MrSantamaria) [inline pkg/e2e/operators/operators.go:477]: Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.

File: pkg/e2e/operators/operators.go [hunk 4]
```diff
@@ -517,7 +517,7 @@ Loop:
 			return nil, err
 		default:
 			if elapsed < timeoutDuration {
-				log.Printf("Waiting %v for %s deployment to exist", (timeoutDuration - elapsed), deploymentName)
+				log.Printf("Waiting %v for %s deployment to exist", timeoutDuration-elapsed, deploymentName)
```
Comment (MrSantamaria) [inline pkg/e2e/operators/operators.go:520]: Could we change this back to having the space? 
Removing the space between variables makes it hard to read, I thought it was one word.

**Review by MrSantamaria — CHANGES_REQUESTED** [review]

File: pkg/e2e/operators/operators.go (hunk 1 above)
Comment (ritmun) [reply pkg/e2e/operators/operators.go:394]: My IDE's auto reformat must've done it - will do

**Comment by MrSantamaria** [comment]
I also don't think we need this anymore. 
From what I validated locally the junit_*.xml is being outputted correctly. 
Refer to this [thread](https://redhat-internal.slack.com/archives/CMK13BP4J/p1673289666907439?thread_ts=1673287242.860939&cid=CMK13BP4J)

**Comment by jbpratt** [comment]
/close

based on https://github.com/openshift/osde2e/pull/1478#issuecomment-1379517516 , please feel free to reopen if this is incorrect!

**Pull request closed without merging**